stores/common_test.go

```diff
@@ -175,9 +175,14 @@ func storeDeleteClient(t tLogger, s Store, clientID string) {
 	}
 }
 
-func storeMsg(t *testing.T, cs *Channel, channel string, data []byte) *pb.MsgProto {
+func storeMsg(t *testing.T, cs *Channel, channel string, seq uint64, data []byte) *pb.MsgProto {
```

Comment:
I wonder if you could not have used a global variable for the sequence that is auto incremented.
For tests that expect sequences to start at a given number, then the test could set the global 
sequence to whatever value at beginning of function.
This is just an idea to reduce the number of changes in the test suite. Now that it is done, it is
kind of a moot point.

Reply:
Issue is that several tests store messages for different channels, so a global variable would result in gaps, unless we maintained a global map containing sequences for each channel.

Reply:
understood. Again, don't bother since you already did the work of changing all tests.